Code.js

```diff
@@ -99,7 +99,7 @@ function buildListHTML(text, delimiter) {
 // buildTableHTML : Content[][] String -> HTML
 // construct an HTML tale give the content (with header row) and modifiers
```

Comment:
Can we convert to JSDoc? Added boilerplate format below for editing
```suggestion
/**
 * Construct an HTML table give the content (with header row) and modifiers
 * 
 * @param {} Content[][]
 * @param {} String
 * @return {} HTML string
 */ 
```